src/pysdk/grvt_api_async.py

```diff
@@ -22,7 +26,7 @@ async def get_all_instruments_v1(
         resp = await self._post(False, self.md_rpc + "/full/v1/all_instruments", req)
         if resp.get("code"):
             return GrvtError(**resp)
-        return types.ApiGetAllInstrumentsResponse(**resp)
+        return from_dict(types.ApiGetAllInstrumentsResponse, resp, Config(cast=[Enum]))
```

Comment:
I'll have to make this the norm in codegen.
`dict -> dataclass` casting doesn't work.